Source/Order.swift

```diff
@@ -111,6 +111,11 @@ final public class Order: NSObject, Codable {
                 }
             }
         }
+        
+        /**
+         A flag indicating whether the customer is new (or not).
+         */
+        var isNew: Bool?
```

Comment:
Is this optional on purpose? If so I assume we want to know when its not sent vs. it being `false`.

Docs say (and the field _is_ optional)
> Value indicating whether or not the customer has previously made a purchase. Pass true if this is their first purchase, otherwise pass false.

Reply:
If we do want null, we should have tests that assert the request does not include the field in that case.

Reply:
@wessmith [This test](https://github.com/button/button-merchant-ios/blob/master/Tests/UnitTests/ReportOrderBodyTests.swift#L53-L80) contains that assertion.

Reply:
Fair although thats not very clear.